hooks/conf_regen/01-yunohost

```diff
@@ -116,8 +116,9 @@ SHELL=/bin/bash
 #   - (sleep random 60 is here to spread requests over a 1-min window)
 #   - if ip.yunohost.org answers ping (basic check to validate that we're connected to the internet and yunohost infra aint down)
 #   - and if lock ain't already taken by another command
-#   - trigger yunohost dyndns update
-*/10 * * * * root : YunoHost DynDNS update; sleep \$((RANDOM\\%60)); ! ping -q -W5 -c1 ip.yunohost.org >/dev/null 2>&1 || test -e /var/run/moulinette_yunohost.lock || yunohost dyndns update >> /dev/null
+#	- check if some domains are flagged as autopush
+#   - trigger yunohost domain dns push --auto
+*/10 * * * * root : YunoHost DynDNS update ; sleep \$((RANDOM\\%60)); ! ping -q -W5 -c1 ip.yunohost.org >/dev/null 2>&1 || test -e /var/run/moulinette_yunohost.lock || !(grep -nR "autopush: 1" /etc/yunohost/domains/*.yml > /dev/null) || yunohost domain dns push --auto >> /dev/null
```

Comment:
There's some ambiguity here : as far as I understand, the new `domain dns push --auto`  is about pushing all domains, not just the "yunohost dyndns" ones.

But this cron job is only created if some dyndns key exist

We probably want to get rid of the legacy `yunohost-dyndns` cron job and have a new `yunohost-autopushdns` (or similar) instead